site/src/App/routes/examples/marketing-banner/marketing-banner.tsx

```diff
@@ -0,0 +1,655 @@
+          of the heading and button is automatically changed to improve the
+          visual contrast. Click the &ldquo;Open in Playroom&rdquo; button on
+          the code pane above to see how this behaves across different colours
+          in different themes.
+        </Text>
+
+        <Step
+          heading="6. Adjust the space in our `Box`"
+          detail={
+            <Stack space="large">
+              <Text component="p">
+                Since adding the background colour we can observe that our
+                container is lacking some much needed breathing room around the
+                content. This is referred to as `padding`
+              </Text>
+              <Text component="p">
+                The <TextLink href="/components/Box">Box</TextLink> component
+                allows you to control the padding using either:
+              </Text>
+              <BulletList>
+                <Bullet>`padding`: all sides equally,</Bullet>
+                <Bullet>`paddingX`: left and right equally,</Bullet>
+                <Bullet>`paddingY`: top and bottom equally, or</Bullet>
+                <Bullet>
+                  `paddingLeft`/`paddingRight`/`paddingTop`/`paddingBottom`:
+                  only.
+                </Bullet>
+              </BulletList>
+              <Text component="p">
+                You can mix and match! So in our banner we will apply
+                &ldquo;small&rdquo; spacing horizontally and
+                &ldquo;xlarge&rdquo; vertically.
+              </Text>
+            </Stack>
+          }
+        >
+          <Box background="brandAccent" paddingX="small" paddingY="xlarge">
+            <Columns space="gutter" alignY="center">
+              <Column width="3/5">
+                <Stack space="large">
+                  <Heading level="1">
+                    Heard about our latest marketing campaign?
+                  </Heading>
+                  <Inline space="none">
+                    <Button weight="weak">Show me</Button>
+                  </Inline>
+                </Stack>
+              </Column>
+              <Column>
+                <Placeholder height={300} label="Marketing illustration" />
+              </Column>
+            </Columns>
+          </Box>
+        </Step>
+
+        <Text component="p">
+          Things are starting to come together now. If you click the &ldquo;Open
+          in Playroom&rsquo; button on the code block above, you&rsquo;ll notice
+          on mobile everything is a squeezed in side-by-side.
+        </Text>
+
+        <Text component="p">We&rsquo;ll address this next.</Text>
+
+        <Step
+          heading="7. Make the layout responsive"
+          detail={
+            <Stack space="large">
+              <Text component="p">
+                One of the features available when using `Columns` is the
+                ability to describe at what point the columns should collapse
```

Comment:
`Columns allows us to define the device size beyond which columns should collapse`?